.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "Ubuntu",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/base:noble",
+	"features": {
+		"ghcr.io/devcontainers/features/rust:1": {
+			"version": "1.91",
+			"profile": "default",
+			"targets": "aarch64-unknown-linux-gnu",
+			"components": "clippy,llvm-tools,rustc-dev,rust-analyzer,rust-src,rustfmt"
+		},
+		"ghcr.io/devcontainers-extra/features/uv:1": {
+			"version": "latest"
+		}
+	},
+	// Uncomment the following when using podman (rootless docker)
+	// Update the uid and gid, to your local users uid and gid (run `echo $UID` and `echo $GID`)
+	// "runArgs": ["--userns=keep-id:uid=1000,gid=1000"],
+	// "containerUser": "vscode",
+	// "updateRemoteUserUID": true,
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
```

Comment:
We could use `features` to e.g. add a `gstreamer` feature to the container, or any other optional dependency set.